docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+## Install with Visual Studio for Mac
+
+Visual Studio for Mac installs the .NET Core SDK when the **.NET Core workload** is selected. To get started with .NET Core development on macOS, see [Install Visual Studio 2019 for Mac](https://docs.microsoft.com/visualstudio/mac/installation?view=vsmac-2019).
+
+[![macOS Visual Studio 2019 for Mac with .NET Core workload feature](media/index/mac-install-selection.png)](media/index/mac-install-selection.png#lightbox)
+
+After you've installed Visual Studio for Mac, create your first app by following the [Get started on macOS](../tutorials/using-on-mac-vs.md) tutorial.
+
+::: zone-end
+
+## Install with Visual Studio Code
+
+Visual Studio Code is a powerful and lightweight source code editor that runs on your desktop. VS Code is available for Windows, macOS, and Linux.
+
+While VS Code doesn't come with .NET Core support, adding .NET Core support is simple.
+
+01. [Download and install Visual Studio Code](https://code.visualstudio.com/Download).
+01. [Download and install the .NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+01. [Install the C# extension from the VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp).
+
+After you've installed .NET Core support for VS Code, create your first app by following the [Using .NET Core in Visual Studio Code](https://code.visualstudio.com/docs/languages/dotnet) tutorial.
+
+::: zone pivot="os-windows"
+
+## Install with PowerShell automation
+
+The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for a non-admin install of the CLI toolchain and the shared runtime. You can download the script from [dotnet-install script reference page](../tools/dotnet-install-script.md).
+
+The script defaults to installing the latest "LTS" version, which is currently .NET Core 2.1. To install the current release of .NET Core (3.0), run the script with the following switch:
+
+```powershell
+dotnet-install.ps1 -Channel LTS
+```
+
+The installer PowerShell script is used in automation scenarios and non-admin installations.
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+## Install with native installers
+
+.NET Core native installers are available for supported Linux distributions/versions. The native installers require admin (sudo) access to the server. The advantage of using a native installer is that all of the .NET Core native dependencies are installed. Native installers also install the .NET Core SDK system-wide.
+
+On Linux, there are two installer package choices:
+
+- Using a feed-based package manager, such as apt-get for Ubuntu, or yum for CentOS/RHEL.
+- Using the packages themselves, DEB or RPM.
+
+## Install with bash automation
```

Comment:
Is there any reason this is not included in the os-mac pivot?  It is a valid mac option and is useful for automation scenarios.